brainframe_qt/ui/resources/video_items/detections/detection_item.py

```diff
@@ -31,7 +31,9 @@ def __init__(self, detection: Detection, *,
             detection, self.draw_color,
             render_config=render_config, parent=self)
 
-        self.detection_track = DetectionTrackItem(track, parent=self)
+        self._detection_track: Optional[DetectionTrackItem] = None
```

Comment:
Should it be a public variable? I'm not sure what the philosophy of the object-oriented rendering is for you

Reply:
I just kinda left all the objects as public members on each of the drawn objects. I'd keep it public

Reply:
Got it. I've changed it back now.